pipelines/optimus/README.md

```diff
@@ -193,23 +197,21 @@ Output files of the pipeline include:
 3. Cell metadata, including cell metrics
 4. Gene metadata, including gene metrics
 
-The following table lists the output files produced from the pipeline. For samples that have sequenced over multiple lanes, the pipeline will output one merged version of each listed file.
+The following table lists the output files produced from the pipeline. For samples that have sequenced over multiple lanes, the pipeline will output one merged version of each listed file. 
 
 | Output Name | Filename, if applicable | Output Type |Output Format |
 | ------ |------ | ------ | ------ | 
 | pipeline_version | | Version of the processing pipeline run on this data | String | 
-| bam | merged.bam | aligned bam | bam |
+| bam | sample_id.bam | Aligned BAM | BAM |
```

Comment:
I think it needs to be clear here that the file isn’t going to be called sample_id.bam it’s <sample_id>.bam based on whatever the input is. Is it <> that usually denotes a variable? I forget. Maybe it’s {} or []